estate/__manifest__.py

```diff
@@ -7,9 +7,16 @@
     """,
     "category": "",
     "data": [
+        #SECURITY
+        'security/res_groups.xml',
         'security/ir.model.access.csv',
-        'views/property_view.xml',
-        'views/menu_items.xml',
+        #VIEWS
+        'views/estate_property_view.xml',
```

Comment:
On utilise le pluriel pour les views par convention
```suggestion
        'views/estate_property_views.xml',
```